packages/http-caching-proxy/README.md

```diff
@@ -89,6 +89,20 @@ await proxy.stop();
 See the auto-generated documentation at
 [loopback.io](https://loopback.io/doc/en/lb4/apidocs.http-caching-proxy.html)
 
+## Alternatives
```

Comment:
IMO, the title is misleading. As we are making clarification that this module is NOT for http req/res replay. Maybe something like `Out of scope use cases`

Reply:
Here is what I meant by this title: "alternative solutions I can use in my integration/acceptance tests"

Would the following title be better?

```suggestion
## Alternative solutions for integration testing
```

Reply:
Maybe `Other considerations for HTTP based integration testing`?

Reply:
Or `Relevant solutions for HTTP based integration testing`?

Reply:
Thank you @raymondfeng for the proposals. I'll go with `Alternative solutions for HTTP-based integration testing`